ts/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@nationalarchives/tdr-generated-graphql",
-  "version": "1.0.41",
+  "version": "1.0.42",
```

Comment:
I think you can just delete this version as it checks the one in the `version.sbt` file. I could be wrong but it's worth a try.